lib/config.js

```diff
@@ -6,12 +6,467 @@ module.exports = {
     default: "ccls",
     order: 0,
   },
-  indexOnChange: {
-    type: "boolean",
-    title: "Index on file change",
-    description: "Tell ccls to index immediately when a file changes.",
-    default: false,
-    order: 10,
+  cclsInit: {
```

Comment:
Maybe move to the end and add `order: 1000`? That way we can keep it expanded by default.